### ISSUE TITLE
Variadic parameters might not need preallocation

### ISSUE THREAD

**Issue by gyuho** [alexkohler/prealloc#4]
```go
a := []int{}
b := []int{1}
c := []int{1,2}
s := []int{}
for _, x := range [][]int{a, b, c} {
    s = append(s, x...)
} 
// s cannot be preallocated
```

But `prealloc` warns about this code?

Thanks.

**Comment by alexkohler** [comment]
Ah, interesting - I think we should probably ignore appends that have variadic parameters?

**Comment by alexkohler** [comment]
Fixed in #18.
